lib/cli/ui/prompt/interactive_options.rb

```diff
@@ -125,10 +155,11 @@ def wait_for_user_input
             when ESC                       ; @state = :esc
             when 'k'                       ; up
             when 'j'                       ; down
+            when '0'                       ; select_n(char.to_i)
```

Comment:
This causes the last option to be picked if multiple is false (instead of being ignored, before).

Reply:
Good catch, fixed and tested non-multiple is ignored and multiple is still working.